### PR TITLE
cloudflare-ddns 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by wyattjoh**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by woodruffw** [comment]
Thanks for your submission. Unfortunately, we can't merge this into `homebrew-core` in its current state - we generally like to see packages gain a bit more attention before adding them to core. It's also generally a good idea to get a user (instead of a maintainer) to submit the package, so that we can gauge demand.

In the mean time, you can always host it in your [own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md). I'll also leave some notes so that you can get an idea of what our review process is like for future submissions.

File: Formula/cloudflare_ddns.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    dir = buildpath/"src/github.com/wyattjoh/cloudflare-ddns"
+    dir.install buildpath.children
+    cd dir do
+      system "go", "build", "-o", bin/"cloudflare-ddns"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    system bin/"cloudflare-ddns", "--help"
```
Comment (woodruffw) [inline Formula/cloudflare_ddns.rb]: We try to use `--help` and `--version` tests as a last resort, preferring tests of more realistic functionality (even just an error message from a command that needs credentials).

If `--help` or `--version` is really all that's practical, testing their output with `shell_output` and `assert_match` is usually a good middle ground.

Comment (wyattjoh) [inline Formula/cloudflare_ddns.rb]: Thanks for the feedback! I had originally copied what `heroku` had done, but I've amended my fork of the formula to match your recommended flow.

If it ever get's popular enough (which, seems unlikely) I'll use this as a feedback.

Comment (wyattjoh) [inline Formula/cloudflare_ddns.rb]: https://github.com/wyattjoh/homebrew-core/commit/5d609f7f017bd51cf4f2d68c0489444bf85ac947

**Pull request closed without merging**